api/transfer.py

```diff
@@ -132,10 +137,26 @@ async def post(self, request) -> JSONResponse:
         request_body = await request.json()
```

Comment:
again complex application logic in an api

Reply:
created a method `unpack_request_body` in the adapter for this